src/amo/components/Errors/ErrorComponent/index.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+import * as React from 'react';
+import makeClassName from 'classnames';
+import NestedStatus from 'react-nested-status';
+
+import Card from 'ui/components/Card';
+
+import './styles.scss';
+
+type Props = {|
+  children?: React.Element<any>,
+  className?: string,
+  code: number,
+  header?: React.Element<any> | string,
+|};
+
+type InternalProps = {|
+  ...Props,
+|};
+
+export default class ErrorComponent extends React.Component<InternalProps> {
+  render() {
+    const { children, className, code, header } = this.props;
+
+    return (
+      <NestedStatus code={code}>
```

Comment:
@bobsilverberg I was trying `code = {code ? code : 400}` to set a default value if `code` is not available. But it gives me a test error saying  - `Unnecessary use of conditional expression for default assignment`
Any idea on how to make it work?

Reply:
It's telling you that because instead of `code = {code ? code : 400}` you can just say `code = {code || 400}`. 😉